styles.css

```diff
@@ -408,15 +421,118 @@ textarea {
   color: var(--clr-white);
 }
 
-.error {
-  position: absolute;
-  left: 10rem;
-  bottom: 1rem;
-  color: rgb(255, 0, 0);
-  font-weight: bold;
+.feature {
```

Comment:
- [ ] Good job so far :+1:  But please make sure that your project fits well to the mobile version to give your user a Good User Experience :smile: 

![mobile_](https://user-images.githubusercontent.com/6615394/211018215-54e60169-0fa9-48bb-bf1a-e83a01e218e9.jpg)